_pkgdown.yml

```
@@ -33,18 +33,18 @@ articles:
     navbar: Covariance Structures
     contents:
       - covariance
-  - title: Degrees of Freedom
-    navbar: Degrees of Freedom
+  - title: Degrees of Freedom and Inference
+    navbar: Degrees of Freedom and Inference
```

```suggestion
  - title: Degrees of Freedom and Testing
    navbar: Degrees of Freedom and Testing
```
(inference is more estimating parameters etc)